src/Components/Subscriptions/Table/index.jsx

```diff
@@ -0,0 +1,203 @@
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(requestData)
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+        window.location.reload();
+        alert('Info successfully sent.');
+      })
+      .catch((error) => {
+        alert(error);
+      });
+  };
+
+  const handleEdit = (id) => {
+    const expandedRowsCopy = [...expandedRows];
+    if (expandedRowsCopy.includes(id)) {
+      expandedRowsCopy.splice(expandedRowsCopy.indexOf(id), 1);
+    } else {
+      expandedRowsCopy.push(id);
+    }
+    setExpandedRows(expandedRowsCopy);
+  };
+
+  return (
+    <section className={styles.classContainer}>
+      <h1>Subscriptions</h1>
+      <div>
+        <button className={styles.button} onClick={handleClick}>
+          New manual subscription
+        </button>
+      </div>
+      <button className={styles.button} onClick={handleAdd}>
+        New automatic subscription (for testing)
+      </button>
+      <table>
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>DAY</th>
+            <th>HOUR</th>
+            <th>TRAINER</th>
+            <th>ACTIVITY</th>
+            <th>DATE</th>
+            <th>MEMBERS</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          {data.map((item) => (
+            <React.Fragment key={item._id}>
+              <tr>
+                <td>{item._id}</td>
+                <td>{item.classes && item.classes.day}</td>
+                <td>{item.classes && item.classes.hour}</td>
+                <td>{item.classes && item.classes.trainer}</td>
+                <td>{item.classes && item.classes.activity}</td>
+                <td>{item.date && item.date.substring(0, 10)}</td>
+                <td>
+                  {item.members &&
+                    item.members.map((member) => (
+                      <div key={member._id}>
+                        {member.firstName &&
+                          member.lastName &&
+                          member.firstName + ' ' + member.lastName}
+                      </div>
+                    ))}
+                </td>
+                <td>
+                  <button className={styles.button} onClick={() => handleEdit(item._id)}>
+                    Edit
+                  </button>
+                  <button className={styles.buttonDelete} onClick={() => handleDelete(item._id)}>
+                    Delete
+                  </button>
+                </td>
+              </tr>
+              {expandedRows.includes(item._id) && (
+                <tr>
+                  <td colSpan="8">
+                    <select
+                      className={styles.selectEdit}
+                      name="editMember"
+                      value={user.idMember}
+                      onChange={onChangeInput}
+                    >
+                      <option value="646014b31c70e12b863ad70a">Juan Manuel Lopez</option>
```

Comment:
same as before, we need to change those hardcode ids to ones that we are sure exist on the DB